RELEASE.md

```diff
@@ -25,32 +25,75 @@ described in the instructions below.
 ## Manual benchmarking
 
 Before releasing, it is important to make sure that the new version doesn't
-induce performance regressions.
+induce performance regressions. The trace replay benchmarks should be used
+for that purpose.
 
-The performances of individual releases of irmin (+index, +repr) are saved in
-`/bench/releases`, inside the benchmarking server.
+The performances of individual releases of irmin are saved inside the
+benchmarking server at `/bench/releases/`:
+```
+.
+└── ncommits_200k
+    ├── irmin2.7.0_index1.4.0_repr0.4.0
+    │   ├── packet_b
+    │   │   ├── stat_summary.json
+    │   │   └── stat_trace.repr
+    │   └── packet_a
+    │       ├── stat_summary.json
+    │       └── stat_trace.repr
+    ├── irmin2.5.4_index1.3.1_repr0.3.0
+    │   ├── packet_b
+    │   │   ├── stat_summary.json
+    │   │   └── stat_trace.repr
+    │   └── packet_a
+    │       ├── stat_summary.json
+    │       └── stat_trace.repr
+    └── irmin2.2.0_index1.2.1
+        ├── packet_b
+        │   ├── stat_summary.json
+        │   └── stat_trace.repr
+        └── packet_a
+            ├── stat_summary.json
+            └── stat_trace.repr
+```
 
-From the benchmarking server, setup the right versions of the right libraries
-and then run Irmin's trace replay:
-```sh
-; export ROOT='/bench/releases/irminX.Y.Z-indexX.Y.Z-reprX.Y.Z'
-; dune exec -- bench/irmin-pack/tree.exe --mode trace --ncommits-trace 100000 /bench/replay-traces/tezos-replayable-v0.repr --verbosity info --path-conversion v0+v1 --artefacts $ROOT --keep-stat-trace
+To test a new release, setup an `c3-small-x86-01` aws instance, fetch a large
+enough replay trace and setup the right versions of the right libraries
+before running the benchmarks.
+
+The benchmark should be run at least twice (to give insights about the noise), 
+the two resulting `stat_trace.repr` files should be copied to 
+`/bench/releases/` using the same naming convention as above. Also set the file
+permissions to read-only using `chmod 444 stat_trace.repr`.
+
+A run of the benchmark is expected to last \~35min.
+
+See [this script](https://github.com/tarides/irmin-tezos/blob/master/bench.sh) for an example of a setup and a bench run.
```

Comment:
This repository is private currently.

Reply:
Yes. I think this is okey since only people with access to that repo play a role in the release processes

Reply:
OK :+1: